provisioning/windows-provision.ps1

```diff
@@ -417,8 +363,6 @@ Get-HotFix | Format-Table -Property HotFixID, Description, InstalledOn
 Write-Host "== Sanity Check of installed tools"
 Write-Host "- Path environment"
 Write-Host (Get-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH).path
-Write-Host '- Sanity check for docker'
-& docker -v ## Client only
 foreach($k in $downloads.Keys) {

```

Comment:
I think that this part could be removed

Reply:
Absolutely. As it should be done on a subsequent PR, let's do it in the updatecli PR ?